src/resolvers/workspace.js

```diff
@@ -320,14 +320,66 @@ module.exports = {
         );
 
         if (!isThereOtherAdmins) {
-          throw new ForbiddenError('You can\'t leave this workspace because you are the last admin');
+          return false;
         }
       }
       await workspaceModel.removeMember(userModel);
 
       return true;
     },
 
+    /**
+     * Mutation in order to leave workspace
+     * @param {ResolverObj} _obj - object that contains the result returned from the resolver on the parent field
+     * @param {string} workspaceId - id of the workspace where the user should be removed
+     * @param {UserInContext} user - current authorized user {@see ../index.js}
+     * @param {ContextFactories} factories - factories for working with models
+     * @return {Promise<boolean>} - true if operation is successful
+     */
+    async deleteWorkspace(_obj, { workspaceId }, { user, factories }) {
+      const workspaceModel = await factories.workspacesFactory.findById(workspaceId);
+
+      if (!workspaceModel) {
+        throw new UserInputError('There is no workspace with provided id');
+      }
+
+      const memberInfo = await workspaceModel.getMemberInfo(user.id);
+
+      /**
+       * Return if user is not Admin.
+       */
+      if (!memberInfo.isAdmin) {
```

Comment:
do we really need this statement? I thought the `@requireAdmin` directive already handle that. Should be tested